scripts/sd_tag_batch.py

```diff
@@ -89,7 +89,8 @@ def run(self, p, in_front, prompt_weight, use_deepbooru, use_weight, no_duplicat
             interrogator = self.filter_words(interrogator, raw_prompt)
         # Remove negative prompt content from interrogator prompt
         if use_negatives:
-            interrogator = self.filter_words(interrogator, p.negative_prompt)
+            raw_negative = p.negative_prompt
```

Comment:
Unsure if there was the possibility of pointer override, so I added a buffer variable. 
I am not sure if this is an issue to think about in Python...